samples/scales/time/line-point-data.html

```diff
@@ -122,10 +122,13 @@
 			window.myLine.update();
 		});
 
-		document.getElementById('addData').addEventListener('click', function() {
+		document.getElementById('addData').addEventListener('click', function() { // TODO : fix issue with addData
```

Comment:
This seems broken

Reply:
See https://github.com/chartjs/Chart.js/issues/5197

Reply:
I would update the comment to link to the issue. It's not clear from the TODO right now what is supposed to be fixed